code/modules/modular_computers/computers/item/tablet.dm

```diff
@@ -331,13 +326,14 @@
 // Round start tablets
 
 /obj/item/modular_computer/tablet/pda
-	icon = 'icons/obj/pda.dmi'
+	icon = 'icons/obj/pda.dmi' //Please do not delete the WIP's. They will make it easier to implement more transparent skins in the future.
```

Comment:
Yes, there's no need since the comment will very likely quickly become obsolete when the WIP icon is used at some point. I feel like it might be forgotten and it doesn't really serve a practical purpose currently since we don't often purge unused icons.

Reply:
okay I'll remove these then!